### ISSUE TITLE
Add ability to force entire picture to display at once when being used as a background

### ISSUE THREAD

**Issue by JakeSteam** [JakeSteam/APODWallpaper#15]
When images are set as a wallpaper, some users might prefer always viewing the whole thing. Currently, the image "fits" itself, losing some of the content off-screen.

**Comment by JakeSteam** [comment]
Looks like the only way to do this is actually resize the bitmap ourselves. This will mean scaling it down to fit inside width / height, then adding extra space to resolve the aspect ratio.

Greyscaling is done inside `ContentItem`, but we don't really want to save the modified image this time. As such, it'll probably be done in `WallpaperHelper`.

**Comment by ThenTech** [comment]
An image seems to fit itself at the left side of the screen (at least for me).
Wouldn't it be possible to just center it?

Instead of `image.Location = (0, 0)`
maybe: `image.Location = (-(image.Width - screen.Width) / 2, -(image.Height - screen.Height) / 2)`

**Comment by JakeSteam** [comment]
@Wosser1sProductions Yes, but currently there's no resizing / placing, the image is just sent to the OS to handle. E.g. on my device it centres it horizontally!

Any solution would be extra resizing work, but it's not especially hard.

**Comment by ThenTech** [comment]
It seems possible to add a Rect with the region in the image that should be visible instead of the `null` parameter [here](https://github.com/JakeSteam/APODWallpaper/blob/bf26c091bb8e8055d93370facb54c98cd82c9c0e/app/src/main/java/uk/co/jakelee/apodwallpaper/helper/WallpaperHelper.kt#L67).

Another way would be to re-adjust the given bitmap with something as described [here](https://stackoverflow.com/a/44682443/6608855).

**Comment by JakeSteam** [comment]
Hmm, good find, although that first approach is only the lock screen. The StackOverflow link is more what I'd be doing!
